<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureEnhancements</class>
 <widget class="QWidget" name="ConfigureEnhancements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="rendererBox">
     <property name="title">
      <string>Renderer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="FMV_hack">
           <property name="text">
            <string>FMV Hack (CoreTiming AddTicks)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="AddTicks">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>77</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Internal Resolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resolution_factor_combobox">
          <item>
           <property name="text">
            <string>Auto (Window Size)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Native (400x240)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2x Native (800x480)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3x Native (1200x720)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4x Native (1600x960)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5x Native (2000x1200)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6x Native (2400x1440)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7x Native (2800x1680)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8x Native (3200x1920)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9x Native (3600x2160)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10x Native (4000x2400)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="toggle_linear_filter">
        <property name="text">
         <string>Enable Linear Filtering</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Post-Processing Shader</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="shader_combobox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Stereoscopy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Stereoscopic 3D Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="render_3d_combobox">
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Side by Side</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Anaglyph</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Depth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="factor_3d">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutBox">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>Screen Layout:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="layout_combobox">
          <item>
           <property name="text">
            <string>Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Single Screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium Screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Large Screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Side by Side</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="swap_screen">
        <property name="text">
         <string>Swap Screens</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="bg_label">
          <property name="text">
           <string>Background Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bg_button">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="utilityBox">
     <property name="title">
      <string>Utility</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QCheckBox" name="toggle_custom_textures">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Replace textures with PNG files.&lt;/p&gt;&lt;p&gt;Textures are loaded from load/textures/[Title ID]/.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Use Custom Textures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="toggle_dump_textures">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dump textures to PNG files.&lt;/p&gt;&lt;p&gt;Textures are dumped to dump/textures/[Title ID]/.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Dump Textures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="toggle_preload_textures">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load all custom textures into memory on boot, instead of loading them when the game requires them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Preload Custom Textures</string>
        </property>
       </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
            <widget class="QLabel" name="label">
              <property name="text">
                <string>Preload Directory</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="preload_dir_path">
              <property name="enabled">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="change_preload_dir">
              <property name="text">
                <string>Change</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>165</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
